src/data-sources/constants.ts

```diff
@@ -15,3 +27,26 @@ export const GOOGLE_SHEETS_API_SCOPES = [
 	'https://www.googleapis.com/auth/drive.readonly',
 	'https://www.googleapis.com/auth/spreadsheets.readonly',
 ];
+
+/**
+ * REST API Source Constants
+ */
+export const AUTH_TYPES = [ 'bearer', 'basic', 'api-key' ] as const;
```

Comment:
We can make this list automattically from `HTTP_SOURCE_AUTH_TYPE_SELECT_OPTIONS`.